Formula/postgresql.rb

```diff
@@ -98,6 +98,9 @@ def post_install
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
+      system "brew", "services", "start", "postgresql"
```

Comment:
Is there any way of doing this without starting the server? If not, I fear this may not be a change we can do.